sw/Core/Src/touch.h

```diff
@@ -753,11 +753,11 @@ void finger_synth_update(int fi) {
 
             int pr = fr->pressure[phase0] * 12;
             int pos = fr->pos[phase0 / 2];
-            pr = (pr && !seq_rhythm.supress) ? randrange(pr, pr+12)-512 : -1024;
+            pr = (pr && !seq_rhythm.supress) ? randrange(pr - 6, pr + 6) : -1024;
             if (pr > 0) {
                 read_from_seq = true;
                 synth_dst_finger->pressure = pr;
-                synth_dst_finger->pos = randrange(pos * 8, pos * 8 + 8);
+                synth_dst_finger->pos = randrange(pos * 8 - 4, pos * 8 + 4);
```

Comment:
i cant remember if pos can go negative is ok... randrange can definitely return -ve